nbs/15_gsa_gls/20-null_simulations/05_gwas/03_10-qc-sex_discrepancy-plot.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>Curious why you did these calculations for female but not male!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Actually, I did it for both extremes of the distribution. According to plink documentation, values close to 1.0 tend to group together in a large cluster (which is the case in this data), so I just picked 0.80. And low values of F are more spread out so I checked different thresholds to see how many samples I would be excluding with different thresholds. Since I don't have much sample size, I was concerned in not excluding too many. I ended up using the standard cutoff values of 0.20 and 0.80.</p>